M2/Macaulay2/m2/localring.m2

```diff
@@ -21,6 +21,9 @@
 --        Legacy code from 2008 is stored in packages/LocalRings/legacy.m2 and is still loaded.
 ---------------------------------------------------------------------------
 
+-- exported in LocalRings.m2
+protect MaximalIdeal
```

Comment:
Does this work?  protecting a symbol in Core, then exporting it in another?

Reply:
Yes.

Reply:
I have more changes for the LocalRings package, but decided not to include anything not absolutely necessary here.

In particular, I'd like to remove `m2/localring.m2` and put it in the LocalRings package.